### ISSUE TITLE
DNR converter crash on uBO rule

### ISSUE THREAD

**Issue by chrmod** [AdguardTeam/tsurlfilter#97]
This rule `@@*$document,csp=worker-src 'none',domain=new.lewd.ninja` from `https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/filters-2020.txt` crashes the converter. 

`abp2drn` converts it to 
```
{"action":{"type":"allow"},"condition":{"requestDomains":["new.lewd.ninja"],"resourceTypes":["main_frame"]},"priority":2000,"id":1},{"action":{"type":"allow"},"condition":{"resourceTypes":["sub_frame"],"initiatorDomains":["new.lewd.ninja"]},"priority":2000,"id":2}]
```
which I'm not sure is correct.

**Comment by 105th** [comment]
@chrmod hello! this bug should be fixed in `v2.1.8`
can you please check?

**Comment by 105th** [comment]
bot has closed this issue according to our CI process, but if this bug is still reproducible - please contact us.
